bin/entrypoint

```diff
@@ -16,12 +16,14 @@ unless push_event.valid?
   exit(1)
 end
 
-puts "Checking if #{file_path} was modified in this push..."
+puts "Checking for modified files in this push that match:", file_paths.map{ |f| " - #{f}" }
 
-if push_event.modified?(file_path)
-  puts "#{file_path} was modified"
-  exit(0)
-else
-  puts "#{file_path} was not modified"
+modified_files = push_event.modified_files(*file_paths)
+
+if modified_files.empty?
+  puts "No files were modified"
   exit(78)
+else
+  puts "The following files were modified:", modified_files.map{ |f| " - #{f}" }
```

Comment:
I really dig logging the specific files that were modified. I haven't seen this way of `puts`-ing out array items onto individual lines, but I like it!

Nitpicky, but what do you think about a more descriptive variable for `f`? Something like `file_name` perhaps?

Reply:
In simple cases like this, I believe a shorter variable names are more readable. It's obvious here that `f` is an element of `modified_files`, so I wouldn't change this.